src/library/basetracksetfeature.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "libraryfeature.h"
```

Comment:
Includes in the same directory should still be relative to the root source folder, i.e. "library/libraryfeature.h"